interfaces/CMakeLists.txt

```diff
@@ -69,7 +62,7 @@ if (WRAP_SOQT)
         ${CMAKE_SOURCE_DIR}/interfaces
         )
 
-    target_link_libraries(soqt SoQt)
+    target_link_libraries(soqt SoQt::SoQt)
```

Comment:
Maybe we should add the `PUBLIC` keyword to all `target_link_libraries`, e.g. `target_link_libraries(soqt PUBLIC SoQt::SoQt)`.